src/main/java/jnr/ffi/LibraryLoader.java

```diff
@@ -474,8 +476,12 @@ private static void addPathsFromFile(Collection<String> paths, File file) {
                 in = new BufferedReader(new FileReader(file));
                 String line = in.readLine();
                 while (line != null) {
-                    // add even if doesn't exist! We're just adding the default paths, we check for validity elsewhere
-                    paths.add(line);
+                    // ignore empties, comments and include directives
+                    if (!line.trim().isEmpty()
+                            && !line.startsWith("#") && !line.startsWith("include ")) {
+                        // add even if doesn't exist! We're just adding the default paths, we check for validity elsewhere
+                        paths.add(line);
```

Comment:
What is this change doing?

Reply:
Ignoring empty lines, comments and `include` directives found in `*.so.conf` files. I would like to one day add support for these `include` directives.
These lines are wasteful and should be ignored